src/tabs.scss

```diff
@@ -8,13 +8,34 @@
     flex-flow: column;
     overflow: hidden;
     position: relative;
-  
-    &__tab {
+
+    &__tab-inner {
       @extend .btn;
       @extend .btn--secondary;
 
+      border: 3px solid transparent;
+      margin: 8px 4px;
+      padding: 6px 10px;
```

Comment:
We have been doing margin/padding with design system variables, but totally understand if that's not possible in this case...

Reply:
Yeah my bad, was supposed to update! Done in [26e35d2](https://github.com/RaspberryPiFoundation/editor-ui/pull/299/commits/26e35d20e04528eb1e75c991d9a99dc3db3bbee5)

I added some custom calculations to match the design more consistently